### ISSUE TITLE
Config file name for php-cs-fixer

### ISSUE THREAD

**Issue by mjanser** [Glavin001/atom-beautify#1537]
The name of the config file for php-cs-fixer is hard-coded as `.php_cs`. This should be extended to also support `.php_cs.dist`.

Reference to code: https://github.com/Glavin001/atom-beautify/blob/cdb52a873d1b025faace99b566cc4f790f061325/src/beautifiers/php-cs-fixer.coffee#L20

Additionally it could be helpful to run php-cs-fixer with the project directory as working directory. This would enable the possibility to use a shell script to do some more advanced things without loosing context.

**Comment by Glavin001** [comment]
Pull Requests welcome!

**Comment by sbine** [comment]
Hey @Glavin001, I'm putting together a PR for this enhancement. Do you have any preference as to whether the tool should assume standard config file names (`.php_cs.dist` & `.php_cs`), or expose a config file setting and leave it up to the user? phpcbf assumes standard file names, but some of the other beautifiers delegate to a setting.

**Comment by Glavin001** [comment]
>  I'm putting together a PR for this enhancement.

Awesome! Thanks @sbine! 

> Do you have any preference as to whether the tool should assume standard config file names (.php_cs.dist & .php_cs), or expose a config file setting and leave it up to the user?

I would like the workflow to be as automatic as possible. If `.php_cs` is the standard file name which PHP-CS-Fixer supports, then please use that as default value for option. 

How about supporting an ordered array of supported file names which would be configurable?
Something like this:

```
php_cs_fixer_config_file_names:
      type: 'array'
      default: ['.php_cs']
      items:
        type: 'string'
      description: "List of config file names which are automatically detected in same directory as the file being beautified."
```

Feel free to add `.php_cs.dist` if that is another standard file.

To be honest, since I do not ever code in PHP and therefore am not a user of the PHP beautifiers, I leave these types of decisions up to the applicable community. So your word holds a lot of weight 😃 . Feel free to discuss amongst yourselves and other PHP users until you are confident it is the best solution going forward. If there are new Issues reported in the future I am very likely to link in anyone who submitted a related Pull Request to answer questions.

> phpcbf assumes standard file names, but some of the other beautifiers delegate to a setting.

I assume you are referring to https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/phpcbf.coffee#L22 ?
I think PHPCBF beautifier should also have a configurable option similar to `php_cs_fixer_config_file_names` (except `phpcbf_config_file_names`).

**Comment by sbine** [comment]
Submitted #1661 which adds a new PHP-CS-Fixer config file setting and automatically scans for `.php_cs.dist` in addition to `.php_cs`.

I don't think it's worth supporting automatic config files beyond the two standard options referenced throughout the [PHP-CS-Fixer docs](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/README.rst), but the new setting does allow for custom config files in unusual locations (e.g. `./standards/.php_cs.dist`) and should therefore satisfy most use cases.

**Comment by Glavin001** [comment]
Published to v0.29.26
